include/drivers/display.h

```diff
@@ -81,6 +81,16 @@ enum display_orientation {
 	DISPLAY_ORIENTATION_ROTATED_270,
 };
 
+/**
+ * @enum refresh_val
+ * @brief Enumeration with possible display refresh
+ *
+ */
+enum refresh_val {
+	PARTIAL_REFRESH = 0,
```

Comment:
PARTIAL_REFRESH what? panel? controller RAM? both? What do you mean by partial? Is this change really necessary? None of the subsystems can deal with it, special handling of this driver is not desired. The same functionality can be achieved by using blanking_off, https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/display/gd7965.c#L120.

Reply:
I was planning to use gd7965.c driver for this[ 2.7inch display ](https://www.waveshare.com/w/upload/2/2d/2.7inch-e-paper-Specification.pdf) . unfortunately it doesn't worked, So I created new driver based on gd7965.c . PARTIAL_REFRESH means display update happens in that defined particular area based on SRAM data and LUT. this new API is introduced because of this particular display's driver IC has 2 commands for updating the display as full and partial with 12h and 16h commands.